events/s3.go

```diff
@@ -51,3 +51,11 @@ type S3Object struct {
 	ETag          string `json:"eTag"`
 	Sequencer     string `json:"sequencer"`
 }
+
+type S3TestEvent struct {
```

Comment:
Missing `Bucket`

sample json from an sqs message
```
{"Service":"Amazon S3","Event":"s3:TestEvent","Time":"2019-02-04T19:34:46.985Z","Bucket":"bmoffatt","RequestId":"7BA1940DC6AF888B","HostId":"q1YDbiaMjllP0m+Lcy6cKKgxNrMLFJ9zCrZUFBqHGTG++0nXvnTDIGC7q2/QPAsJg86E8gI7y9U="}
```

Reply:
Whoops, thanks for fixing!